content/settings-template.xml

```diff
@@ -83,8 +83,8 @@
 		<Setting name="DESTROY_TOOL" type="list"> X </Setting>
 		<Setting name="ROTATE_RIGHT" type="list"> PERIOD </Setting>
 		<Setting name="ROTATE_LEFT" type="list"> COMMA </Setting>
-		<Setting name="SPEED_UP" type="list"> PLUS ; EQUALS </Setting>
-		<Setting name="SPEED_DOWN" type="list"> MINUS </Setting>
+		<Setting name="SPEED_UP" type="list"> PLUS ; EQUALS ; KP_PLUS</Setting>
+		<Setting name="SPEED_DOWN" type="list"> MINUS ;  ; KP_MINUS</Setting>
```

Comment:
is here not an ";" to much?

Reply:
The second binding for SPEED_DOWN is left empty on purpose.
If the second binding would be used for KP_MINUS then the binding for KP_MINUS would be in the setting menu, but KP_PLUS wouldn't.
I thought it would be more consistent to keep both out of the settings menu.
This is especially because users can not bind KP_MINUS from the settings menu, so if they would change it they could not get it back.